Library/Formula/libstfl.rb

```diff
@@ -18,6 +18,10 @@ class Libstfl < Formula
   patch :DATA
 
   def install
+    if Formula["ruby"].installed?
```

Comment:
> yes, as I mentioned here:

Aye, I recalled you mentioning it.

> One, when we don't add the path directly (as in this PR) and we don't add it indirectly (by adding the dependency), how is it finding out about the brew ruby at all? 

`superenv` isn't perfect; we don't _yet_ guard `lib` or `include`, just `bin`, so it's possible although uncommon where a dependency isn't explicitly specified and it exists in two places for a build to find one element from one location _but_ another from elsewhere.

I notice from the logs it's calling the system Ruby headers:
```
-I/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/include/ruby-2.0.0/universal-darwin15 -I/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/include/ruby-2.0.0/ruby/backward -I/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/include/ruby-2.0.0
```
But the `-lruby` isn't guarded by a prefix:
```
clang called with: -pthread -dynamic -bundle -o stfl.bundle stfl_wrap.o -L. -L/usr/local/Cellar/libstfl/0.22/lib -pthread ../libstfl.a -lncurses -lruby -liconv
```
And `superenv` is chucking `/usr/local/lib` in the search path:
```
superenv added:    -pipe -w -Os -march=native -isystem/usr/local/include -isystem/usr/include/libxml2 -isystem/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -L/usr/local/lib
```
I suspect the end result is that it's using system Ruby headers and Homebrew Ruby libraries.